clang/lib/AST/ASTContext.cpp

```diff
@@ -451,8 +451,17 @@ const RawComment *ASTContext::getRawCommentForAnyRedecl(
     if (LastCheckedRedecl) {
       if (LastCheckedRedecl == Redecl) {
         LastCheckedRedecl = nullptr;
+        continue;
```

Comment:
The fix for #108145 has merged now, so let's go ahead and drop this hunk

Reply:
Done.